stdlib/public/SwiftShims/LibcOverlayShims.h

```diff
@@ -116,4 +116,19 @@ int static inline _swift_stdlib_openat(int fd, const char *path, int oflag,
 #pragma clang assume_nonnull end
 #endif
 
+#if defined(__OpenBSD__)
+#include <stdio.h>
```

Comment:
Let's keep the includes up top with the others.

Reply:
Done.